build/dependencies.yaml

```diff
@@ -117,7 +117,7 @@ dependencies:
   # Golang
   # TODO: this should really be eliminated and controlled by .go-version
   - name: "golang: upstream version"
-    version: 1.25.3
+    version: 4
```

Comment:
Not

```suggestion
    version: 1.25.4
```
?

Reply:
https://github.com/kubernetes/kubernetes/pull/135201